### PR TITLE
[risk=no] Remove scrolling container

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. Removed content-area which gets the inner scroll container.
2. Changed gender pie charts to bar.
3. Addresses DB-161, DB-168, DB-159, DB-169

### CODE REVIEW

File: api/db-cdr/changelog-schema/db.changelog-24.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet author="srushtigangireddy" id="changelog-24">
+    <validCheckSum>ANY</validCheckSum>
+    <addColumn tableName="survey_module">
+      <!-- To display the surveys in the release order -->
```
Comment (calbach) [inline api/db-cdr/changelog-schema/db.changelog-24.xml]: Is this related to this PR?

Comment (SrushtiGangireddy) [reply api/db-cdr/changelog-schema/db.changelog-24.xml]: This pr addresses DB-161, DB-168, DB-159 and DB-169.

**Comment by SrushtiGangireddy** [comment]
These are little prs so wanted to put them together.

File: public-ui/src/app/data-browser/concept-charts/concept-charts.component.ts [hunk 2]
```diff
@@ -87,6 +77,30 @@ export class ConceptChartsComponent implements OnInit, OnDestroy {
       this.loadingStack.pop();
       }));
   }
+  public fetchMeasurementGenderResults() {
+    if (this.analyses) {
```
Comment (calbach) [inline public-ui/src/app/data-browser/concept-charts/concept-charts.component.ts]: nit: Avoiding conditional nesting where possible generally makes things more readable, so I would recommend instead:

```
if (!this.analyses) {
  return;
}
```

File: public-ui/src/app/views/ehr-view/ehr-view.component.ts [hunk 3]
```diff
@@ -95,9 +108,13 @@ export class EhrViewComponent implements OnInit, OnDestroy {
       this.prevSearchText = '';
     }
     this.searchText.setValue(this.prevSearchText);
-    const obj = localStorage.getItem('ehrDomain');
+    const obj = localStorage.getItem( 'ehrDomain');
```
Comment (calbach) [inline public-ui/src/app/views/ehr-view/ehr-view.component.ts]: nit: revert

File: public-ui/src/app/views/ehr-view/ehr-view.component.ts [hunk 4]
```diff
@@ -182,16 +192,37 @@ export class EhrViewComponent implements OnInit, OnDestroy {
...
+    this.showAgeGraph = false;
+    this.showSourcesGraph = false;
+    if (g === 'Gender') {
+      this.showGenderGraph = true;
+    } else if (g === 'Age') {
+      this.showAgeGraph = true;
+      this.showMeasurementGenderBins = false;
+    } else if (g === 'Sources') {
+      this.showSourcesGraph = true;
+      this.showMeasurementGenderBins = false;
```
Comment (calbach) [inline public-ui/src/app/views/ehr-view/ehr-view.component.ts]: nit: why not just set this to false at the start, then set to true under `g === 'Gender'`?
